app/controllers/energy_tariffs/energy_tariffs_controller.rb

```diff
@@ -4,23 +4,30 @@ class EnergyTariffsController < ApplicationController
     include EnergyTariffable
 
     load_and_authorize_resource :school
+    load_and_authorize_resource :school_group
     load_and_authorize_resource :energy_tariff
     before_action :admin_authorized?, if: :site_settings_resource?
     before_action :load_site_setting, if: :site_settings_resource?
     before_action :set_breadcrumbs
 
     def index
+      authorize! :manage, @school.energy_tariffs.build if @school
+      authorize! :manage, @school_group.energy_tariffs.build if @school_group
+
       if @school
         @electricity_meters = @school.meters.electricity
         @electricity_tariffs = @school.energy_tariffs.electricity.by_start_date.by_name
         @gas_meters = @school.meters.gas
         @gas_tariffs = @school.energy_tariffs.gas.by_start_date.by_name
       end
+      @site_setting = SiteSettings.current if @school_group
```

Comment:
With this set, then we end up seeing duplicate tariffs on the school group page:

```
    <%= render 'energy_tariffs_tab_school_groups' if @school_group %>
    <%= render 'energy_tariffs_tab_site_settings' if @site_setting %>
```

Means we see the school groups tariffs, then all of the site settings. The latter should only be in the default view.

Reply:
:+1: done